test/EventStore.Client.Streams.Tests/DependencyInjectionTests.cs

```diff
@@ -12,13 +12,41 @@ public class DependencyInjectionTests {
 				.BuildServiceProvider()
 				.GetRequiredService<EventStoreClient>();
 
+		[Fact]
+		public void RegisterWithServiceProvider() =>
+			new ServiceCollection()
+				.AddEventStoreClient(provider => { })
```

Comment:
Is this exerising the intended overload?

Reply:
No, it was using the wrong overload - I've pushed a fix. Good catch 😊 

I've also renamed some of the tests to make it more obvious which overload they're testing. I hope that's okay.